test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -73,7 +82,7 @@ if [ ! ${ZCC_TEST_NAME[$FRAMEWORK]} ]; then
     echo "$FRAMEWORK is not a valid entry in list of frameworks"
     exit 1
 fi
-if [[ ${FRAMEWORK} == "tensorflow2" ]]; then
+if [[ ${FRAMEWORK} == "tensorflow2" || $TF_VERSION != "2.3.0" ]]; then
```

Comment:
This if-condition exists so that tensorflow 2.x will be tested with `test_tensorflow2_gradtape_integration.py`.

TF_VERSION != "2.3.0" will translate to `True` for MXNet and PyTorch, as well as TensorFlow <=1.15.

Reply:
You're right. I'll change the boolean